javascript/javascript2/week1/SpiritAnimalGenerator/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Spirit Animal Generator</title>
+  
+</head>
+<body>
+   
+    <h1>Spirit Animal Generator</h1> 
+   
+    <div>
+    <form action="/action_page.php">     
+        <label id="lbltitle">You can choose here!</label>
+   
+        <select id="selectEvent" onchange="onSelectChange()">
+            <option value="Select">Select</option>
+            <option value = "Click" id="clickMe">Click Button</option>
+            <option value = "Hover" id="hoverme">Hover on Button</option>
+        </select>
+
+      </form> 
+    </div>
+    <script src="spiritAnimalGenerator.js"></script>   
```

Comment:
Just curious. Why didn't you create some template html, such as input and dropdown here?

Reply:
I didn't understand what you meant here.

Reply:
Thanks for the comments. Can you help me with an example?I am not sure if I'm getting this right.